lib/errors.ts

```diff
@@ -122,6 +134,23 @@ const messages: {
 
 	EACCES: (e) => messages.EPERM(e),
 
+	BalenaSettingsPermissionError: () => {
```

Comment:
> Example of test:
> [...]
> Additional information may be available with the `--debug` flag.
> For further help or support, visit: [...]

The part above that reads _"Additional information ..."_ indicates that the CLI is treating this error as "unexpected" (as if improperly handled), and reporting it to Sentry.io. Let's avoid this by declaring this error as "expected". I think it is just a matter of adding the following entry to the following array -- find it in this same file:

```
const EXPECTED_ERROR_REGEXES = [
	/^BalenaSettingsPermissionError/,
```

Reply:
Sweet. NPM fixed and now it's an expected error.

New Message:
```
Error reading data directory: "/home/dev/.balena"

This error usually indicates that the user doesn't have permissions over that directory,
which can happen if balena CLI was executed as the root user.

Try resetting the ownership by running: `sudo chown -R $(whoami) /home/dev/.balena`
```